### ISSUE TITLE
Many files in repo

### ISSUE THREAD

**Issue by richelbilderbeek** [pratikunterwegs/tres-tidy-tutorial#44]
From Pratik, 10 July 2020 20:35:

> Some guesses on why the repo has files everywhere.
>
>My understanding is that Travis is building the book in the top folder, but I'm not sure why it's also going into the docs folder. It could be that my render_book script works properly only locally. I'm not sure how to determine that.
>
>My solution would be to turn off Travis, and build the book locally. If it is doing something specific other than that, can the book rendering be fixed first?

**Comment by richelbilderbeek** [comment]
> My understanding is that Travis is building the book in the top folder, but I'm not sure why it's also going into the docs folder. It could be that my render_book script works properly only locally. I'm not sure how to determine that.

A nice guess, but definitely incorrect, as Travis does not push files at all. As can be seen [in the Travis CI file](https://github.com/pratikunterwegs/tres-tidy-tutorial/blob/master/.travis.yml#L15), all it does is call `render_books.sh` ...

> My solution would be to turn off Travis, and build the book locally. If it is doing something specific other than that, can the book rendering be fixed first?

... from which we conclude this is not a solution.

I am sure that **I** built the files locally, on my Ubuntu 20.04 (Focal Fossa) distro. Rebuilding on another machine may bring back things as desired?

**Comment by richelbilderbeek** [comment]
I will move the pictures I use to the `data` folder, as I am unsure what the desired location is. It will definitely reduce the number of files in the root.

**Comment by richelbilderbeek** [comment]
Taking a look at [an earlier version](https://github.com/pratikunterwegs/tres-tidy-tutorial/tree/4d6a7847db6c23111f12ec1d03f2430dd808c11f), I see it was mainly images and HTML files in the root. Regarding the HTML files: these are removed now (no idea why they got there!) and I also added `*.html` to `.gitignore` to accidentally add these again. The images I moved to the `data` folder.

As the number of files in the root is to the original level again, I close this Issue :+1:
